Makefile

```diff
@@ -52,9 +53,12 @@ release: setup-release
 install:
 	@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
 
-dev:
+remove:
 	-@luarocks remove kong
-	@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
+
+reinstall: remove install
+
+dev-only:
```

Comment:
I think having `make dev` and `make dev-only` would be very confusing. I suggest renaming `dev-only` to `dependencies`.